config/c-code/tests.yml.j2

```diff
@@ -98,21 +98,33 @@ jobs:
 {% if with_future_python and with_pypy %}
       - name: Install Build Dependencies (PyPy2)
         if: >
-           startsWith(matrix.python-version, 'pypy-2.7')
+          startsWith(matrix.python-version, 'pypy-2.7')
         run: |
           pip install -U pip
           pip install -U setuptools wheel twine "cffi != 1.15.1"
       - name: Install Build Dependencies (other Python versions)
         if: >
-           !startsWith(matrix.python-version, 'pypy-2.7')
+          !startsWith(matrix.python-version, 'pypy-2.7')
 {% else %}
       - name: Install Build Dependencies
 {% endif %}
         run: |
           pip install -U pip
           pip install -U setuptools wheel twine cffi
 
-      - name: Build %(package_name)s
+{% for kind in ('Python 3.10 on MacOS', 'all other versions') %}
+      - name: Build %(package_name)s (%(kind)s)
+  {% if kind == 'Python 3.10 on MacOS' %}
+        if: >
+          startsWith(runner.os, 'Mac')
+          && startsWith(matrix.python-version, '3.10')
+        env:
+          _PYTHON_HOST_PLATFORM: macosx-11-x86_64
+  {% else %}
+        if: >
```

Comment:
If I see it correctly, this condition is not needed: if it is true, nothing happens.

Reply:
If it is true the `run` command in line 116+ is executed, see https://github.com/zopefoundation/zope.security/pull/87/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR156-R177 for an example of generated code.